CMakeLists.txt

```diff
@@ -25,11 +25,6 @@ endif()
 
 include(${PROJECT_SOURCE_DIR}/toolchains/utils.cmake)
 
-# cmake output configuration
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
Ah I see, that is a bit inconvenient. It turns out that these specific lines don't actually impact the Android build, so I'll revert this change.